int16 address
int8 command
uint8 value
int8 retries
int8 priority
---
int8 status
